### PR TITLE
update

### PR DESCRIPTION
**Pull request by dpanshug**

### CODE REVIEW

File: data-grid.adoc [hunk 1]
```diff
@@ -54,9 +54,14 @@ TIP: Infinispan recommends the `application/x-protostream` media type. Protobuf
 Expiration removes entries based on a lifespan or maximum idle time. Expiration takes effect across Infinispan clusters.
 
 Lifespan:: The maximum amount of time, in milliseconds, that cache entries can exist. If you set the value to -1, entries never expire.
+
+TIP: Using Lifespan expiration causes performance issue for `get()` update all instance, lifespan is better if very close.
```
Comment (dvagnero) [inline data-grid.adoc:58]: Both TIPS need clarification from Wolf.

**Comment by domiborges** [comment]
@dpanshug Do you want to continue working in this repo? I would prefer moving the cheat sheet in infinispan/documentation.  I want to add some useful links to the docs and in infinispan repo I could use the attributes for that. Let me know and I can move it there.

**Comment by dpanshug** [comment]
> @dpanshug Do you want to continue working in this repo? I would prefer moving the cheat sheet in infinispan/documentation. I want to add some useful links to the docs and in infinispan repo I could use the attributes for that. Let me know and I can move it there.

Yes, please move this repo to a better place :100: .

**Comment by dpanshug** [comment]
@dvagnero is this PR changes available in https://github.com/infinispan/infinispan/pull/10392 ?

**Comment by domiborges** [comment]
yes, I copied this file

**Pull request closed without merging**